core/sql/regress/tools/runregr_udr.ksh

```diff
@@ -851,9 +851,9 @@ if [ $LINUX -ne 0 ]; then
 fi
 
```

Comment:
Thanks, now I understand why you are suggesting this. I'll push an update shortly.